### PR TITLE
Fix dirname in applyRuntimeQueryFile

### PR DESCRIPTION
**Pull request by MaciejBaj**

### CODE REVIEW

File: helpers/database.js [hunk 1]
```diff
@@ -103,6 +102,7 @@ function Migrator (pgp, db) {
 	};
 
 	this.applyRuntimeQueryFile = function (waterCb) {
+		var dirname = process.env.NODE_ENV === "test" ? path.join(__dirname, '..') : __dirname;
```
Comment (karmacoma) [inline helpers/database.js]: Single quotes.

File: test/common/initModule.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ var database = require(path.join(dirname, '/helpers', 'database.js'));
 var genesisblock = require(path.join(dirname, '/genesisBlock.json'));
 var Logger = require(dirname + '/logger.js');
 
+process.env.NODE_ENV = "test";
```
Comment (karmacoma) [inline test/common/initModule.js]: Single quotes.

File: helpers/database.js [hunk 3]
```diff
@@ -103,6 +102,7 @@ function Migrator (pgp, db) {
 	};
 
 	this.applyRuntimeQueryFile = function (waterCb) {
+		var dirname = require('path').basename(__dirname) === 'helpers' ? path.join(__dirname, '..') : __dirname;
```
Comment (Isabello) [inline helpers/database.js]: While this solution isnt the prettiest its a better option than using more NODE_ENVs

Comment (MaciejBaj) [inline helpers/database.js]: we have to remember not to name the release with tag 'helpers'

**Pull request merged**